src/lib/maybe-hydrate.js

```diff
@@ -91,6 +91,18 @@ module.exports = function maybeHydrate (inventory, callback) {
             }
             else callback()
           },
+          function _deno(callback) {
+            if(inv.lambdasBySrcDir[path] !== undefined) {
```

Comment:
This check is unnecessary, this function only got called on Lambdas that exist in the project inventory

Reply:
Ah.. the tests fail without that block

```
/Users/martinhicks/Projects/sandbox/src/lib/maybe-hydrate.js:95
            let isDenoRuntime = (inv.lambdasBySrcDir[path].config.runtime === 'deno')
                                                           ^

TypeError: Cannot read property 'config' of undefined
```

![Screenshot 2021-05-28 at 13 28 48](https://user-images.githubusercontent.com/785770/119984222-0d6c7a00-bfb9-11eb-90b0-7b9553cc42f0.png)

Reply:
OK, happens when the path is the `shared` or `views` dir path. Have added that to the `isDenoRuntime` condition